grpc-ballerina/build.gradle

```diff
@@ -106,7 +106,6 @@ task copyToLib(type: Copy) {
 }
 
 task unpackJballerinaTools(type: Copy) {
-    dependsOn(copyToLib)
```

Comment:
@azinneera we need to enable this. That is the cause of the build failure.

Reply:
Has happened by mistake. Readded the line